test/mgh07.jl

```diff
@@ -0,0 +1,32 @@
+# MGH problem 7 - Helical valley function
+#
+#  Source:
+#  J. J. Moré, B. S. Garbow and K. E. Hillstrom
+#  Testing Unconstrained Optimization Software
+#  ACM Transactions on Mathematical Software, 7(1):17-41, 1981
+#
+# A. Montoison, Montreal, 05/2018.
+
+export mgh07
+
+"Helical valley function"
+function mgh07()
+
+  nls  = Model()
+  x0   = [-1.0,  0.0,  0.0]
+  lvar = [-Inf, -Inf, -Inf]
+  uvar = [ Inf,  Inf,  Inf]
+  @variable(nls, lvar[i] <= x[i=1:3] <= uvar[i], start = x0[i])
```

Comment:
Is it necessary to give the bounds if there aren't any?

Reply:
No, I will delete lvar and uvar for the two models.